Sources/Network/HTTPS/DefaultHttpRequester.swift

```diff
@@ -93,55 +106,61 @@ extension DefaultHttpRequester {
         }
 
         /// Compare pinned & server public keys
-        let matches : [ChainOfTrustKey]
-        matches = trust.publicKeyTrustChain.enumerated().map { chain -> ChainOfTrustKeyMatch in
+        let matches: [ChainOfTrustKey]
+        let trustChainEnumerated = trust.publicKeyTrustChain.enumerated()
+        matches = trustChainEnumerated.map { chain -> ChainOfTrustKeyMatch in
             let serverCertificateKey = chain.element
             let match = pinnedKeys.contains(serverCertificateKey)
             return (match: match, index: chain.offset, key: serverCertificateKey)
-        }.filter {
+        }
+        .filter {
             $0.match
-        }.map {
+        }
+        .map {
             (index: $0.index, key: $0.key)
         }
```

Comment:
matches = trustChainEnumerated
            .map { chain -> ChainOfTrustKeyMatch in
                let serverCertificateKey = chain.element
                let match = pinnedKeys.contains(serverCertificateKey)
                return (match: match, index: chain.offset, key: serverCertificateKey)
            }
            .filter { $0.match }
            .map { (index: $0.index, key: $0.key) }

this will be a violation, top one not sure

Reply:
using second suggestion, passes swiftlint.